api/Node.json

```
@@ -122,7 +122,7 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": true
```

Out of curiosity, I went looking to see when this was added and [`baseURI` is pretty darn old](https://www-archive.mozilla.org/start/1.0/guide/product.html#standards). I feel pretty confident that Firefox can be set to version 1 (and firefox_android to version 4).

```suggestion
              "version_added": "1"
```